wagtail/admin/templates/wagtailadmin/shared/page_status_tag_new.html

```diff
@@ -10,7 +11,7 @@
 {% if page.live and page.url is not None %}
     {% test_page_is_public page as is_public %}
 
-    <a href="{{ page.url }}" target="_blank" rel="noreferrer"
+    <a href="{% pageurl page as page_url %}" target="_blank" rel="noreferrer"
```

Comment:
```suggestion
    <a href="{% pageurl page %}" target="_blank" rel="noreferrer"
```

Same here - we do not need to use the `as` if just putting the result straight into the template output.